### PR TITLE
docs: Verifying that your tools are using Podman

### LINKED ISSUE #1036: docs: add guide on how to work if you used to have docker installed
### Is your enhancement related to a problem? Please describe

People may have used docker in the past, but after installing Podman Desktop, they expect to have all the tools working or expect that some command lines work.

### Describe the solution you'd like

List all steps to ensure you can work with Podman but make it as if Docker was installed

- [x] https://github.com/containers/podman-desktop/pull/1088
- [x] https://github.com/containers/podman-desktop/pull/1135
- [x] https://github.com/containers/podman-desktop/pull/1143
- [x] https://github.com/containers/podman-desktop/pull/1144
- [x] https://github.com/containers/podman-desktop/pull/1146
- [x] https://github.com/containers/podman-desktop/pull/1152
- [x] https://github.com/containers/podman-desktop/pull/1162

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

fixes: https://github.com/containers/podman-desktop/issues/1036

Signed-off-by: Fabrice Flore-Thébault <ffloreth@redhat.com>

### Screenshot/screencast of this PR

Screenshot from 2023-01-10 14-40-45

### What issues does this PR fix or reference?

fixes: https://github.com/containers/podman-desktop/issues/1036

### CODE REVIEW

File: website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [The `DOCKER_HOST` environment variable is set](using-the-docker_host-environment-variable)
+* [On macOS, the `podman-mac-helper` service is running](using-podman-mac-helper)
+* [Podman is emulating Docker CLI](emulating-docker-cli-with-podman)
+
+#### Procedure
+
+1. The Docker socket replies successfully:
+
+    ```shell-session
+    $ curl /var/run/docker.sock
```
Comment (benoitf) [inline website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md]:
```suggestion
    $ curl --unix-socket /var/run/docker.sock "http:/v1.41/containers/json?all=true"
```

File: website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* Docker service is stopped, or not installed.
+* [Saved containers are imported to Podman](importing-saved-containers)
+* [The `DOCKER_HOST` environment variable is set](using-the-docker_host-environment-variable)
+* [On macOS, the `podman-mac-helper` service is running](using-podman-mac-helper)
+* [Podman is emulating Docker CLI](emulating-docker-cli-with-podman)
+
+#### Procedure
+
+1. The Docker socket replies successfully:
```
Comment (benoitf) [inline website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md]:
```suggestion
1. The Docker socket replies successfully for listing all containers:
```

**Review by benoitf — APPROVED** [review]

File: website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+1. The Docker socket replies successfully for listing all containers:
+
+    ```shell-session
+    $ curl --unix-socket /var/run/docker.sock "http:/v1.41/containers/json?all=true"
+    ```
+
+2. Podman commands run successfully when redirected to the Docker socket:
+
+    ```shell-session
+    $ CONTAINER_HOST=/var/run/docker.sock podman ps
```
Comment (benoitf) [inline website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md]: On Windows the socket is at `npipe:////./pipe/docker_engine` (there is no /var/run folder on Windows)

**Review by slemeur — APPROVED** [review]

File: website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* [On macOS, the `podman-mac-helper` service is running](using-podman-mac-helper)
+* [Podman is emulating Docker CLI](emulating-docker-cli-with-podman)
+
+#### Procedure
+
+1. The Docker socket replies successfully for listing all containers:
+
+    ```shell-session
+    $ curl --unix-socket /var/run/docker.sock "http:/v1.41/containers/json?all=true"
+    ```
```
Comment (themr0c) [inline website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md]:
```suggestion

    * On Linux and macOS
    
      ```shell-session
      $ curl --unix-socket /var/run/docker.sock "http:/v1.41/containers/json?all=true"
      ```

    * On Windows
    
      ```shell-session
      $ curl --unix-socket npipe:////./pipe/docker_engine "http:/v1.41/containers/json?all=true"
      ```

```

File: website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+2. Podman commands run successfully when redirected to the Docker socket:
+
+    ```shell-session
+    $ CONTAINER_HOST=/var/run/docker.sock podman ps
+    ```
```
Comment (themr0c) [inline website/docs/migrating-from-docker/verifying-your-tools-are-using-podman.md]:
```suggestion
    * On Linux and macOS
    
      ```shell-session
      $ CONTAINER_HOST=/var/run/docker.sock podman ps
      ```

    * On Windows
    
      ```shell-session
      $ CONTAINER_HOST=npipe:////./pipe/docker_engine podman ps
      ```

```

**Comment by benoitf** [comment]
do you think we could use tabs https://docusaurus.io/docs/markdown-features/tabs

for Windows or Linux/macOS ?

**Comment by themr0c** [comment]
Nice idea. I will have a look into tabs.

**Pull request merged**